code/modules/mob/living/basic/space_fauna/netherworld/creature.dm

```diff
@@ -37,9 +35,15 @@
 	var/datum/callback/health_changes_callback = CALLBACK(src, PROC_REF(health_check))
 	AddElement(/datum/element/swabable, CELL_LINE_TABLE_NETHER, CELL_VIRUS_TABLE_GENERIC_MOB, 1, 0)
 	AddComponent(/datum/component/damage_buffs, health_changes_callback)
-	var/datum/action/innate/creature/teleport/teleport = new(src)
+	var/datum/action/cooldown/spell/jaunt/creature_teleport/teleport = new(src)
 	teleport.Grant(src)
 
+/mob/living/basic/creature/Destroy()
+	if(nest)
+		nest.spawned_mobs -= src
+		nest = null
+	return ..()
```

Comment:
How does this interact with #73645? This isn't in your PR Body either, so I'm confused why it's here. If you need #73645 merged before this one, lmk

Reply:
This is actually just moved from elsewhere in the file, it's not new.
And it moved around because I removed it, then realised it was supposed to be there and put it back, then got annoyed that it needed to be there and opened a different PR 😅

Reply:
i see